sdk/mobile-center-distribute/src/main/java/com/microsoft/azure/mobile/distribute/Distribute.java

```diff
@@ -1138,15 +1146,8 @@ private void showDownloadProgress() {
      */
     private synchronized void hideProgressDialog() {
         if (mProgressDialog != null) {
-            final Dialog progressDialog = mProgressDialog;
+            mProgressDialog.hide();
             mProgressDialog = null;
```

Comment:
This is main thread here so I don't need the handler. I use the handler in another place to ensure that, see patch.